src/io/flutter/editor/FlutterPubspecNotificationProvider.java

```diff
@@ -55,13 +55,12 @@ public EditorNotificationPanel createNotificationPanel(@NotNull VirtualFile file
       return null;
     }
 
-    final Module module = ModuleUtilCore.findModuleForFile(file, project);
-    if (module == null || !FlutterModuleUtils.usesFlutter(module)) {
+    // Check that this pubspec file declares flutter
+    if (!PubRoot.declaresFlutter(file)) {
```

Comment:
Perhaps put this method in `FlutterUtils`?  (We've got a bunch of similar methods there and I think it's a slightly more obvious place to look.)

Reply:
Done.  I had to also moved the isFlutterPlugin method, since I didn't want 2 versions of loadPubspecInfo.